src/convertToObject.js

```diff
@@ -3,16 +3,30 @@
 /**
  * Implement convertToObject function:
  *
- * Function takes a string with styles (see an example in [stylesString.js](./stylesString.js))
+ * Function takes a string with styles
+ * (see an example in [stylesString.js](./stylesString.js))
  * and returns an object where CSS properties are keys
- * and values are the values of related CSS properties (see an exampl in [test file](./convertToObject.test.js))
+ * and values are the values of related CSS properties
+ * (see an exampl in [test file](./convertToObject.test.js))
  *
  * @param {string} sourceString
  *
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesObject = {};
+
+  sourceString
+    .split(';')
+    .map((style) => style.trim())
+    .filter((style) => style.length > 0)
+    .forEach((style) => {
+      const styleArr = style.split(':');
+
+      stylesObject[styleArr[0].trim()] = styleArr[1].trim();
```

Comment:
```suggestion
      const [styleKey, styleValue] = style.split(':');

      stylesObject[styleKey.trim()] = styleValue.trim();
```

Reply:
For better readability.

Reply:
@SalehBUD let's apply @nurdanokcu's suggestion - it's a common good practice.